site/specs/numbers.json

```diff
@@ -1421,7 +1421,7 @@
           "/accounts"
         ],
         "description": "<p>Retrieves bulk port-in orders for the specified accountId.\nA maximum of 1,000 orders can be retrieved per request. If no date range or specific query parameter is provided, the order results will be limited to the last two years.</p>\n",
-        "operationId": "GET /accounts/{accountId}/bulkPortins",
```

Comment:
I was thinking of leaving this field alone and adding a `summary` field with the friendly name.  @hamermike is this the right thing to do?  What will be used for the SDK method name?

Reply:
I'd include a short summary like you mentioned and update the `operationId` to be something like... `GetBulkPortins` in this instance. You can see what messaging / voice do as an example. https://github.com/Bandwidth/api-docs/blob/main/site/specs/messaging.json